README.md

```diff
@@ -1,57 +1,49 @@
-Datapunt panorama API
-======================
+## Datapunt Panorama-API
```

Comment:
readme in english please

Reply:
README was in het Nederlands, en ik laat 'm voorlopig in 't Nederlands.

Reply:
ik zie geen enkele test. je kunt je nieuwe filters toch op zijn minst even testen..